ucsd-fair-data-informatics.md

```diff
@@ -0,0 +1,67 @@
+# UCSD Fair Data Informatics Lab - Neuroscience Information Framework
+## Description
+The University of California, San Diego (UCSD) Fair Data and Informatics Laboratory (FDI Lab) supports open science and is 
+home to the Neuroscience Information Framework (NIF). NIF has extensive experience in community
+engagement, collaborative standards building, effective processes for sharing, federating, and
+integrating neuroscience data, information, and knowledge.
+
+# Roles
+[Principal Investigator] Maryann Martone (maryann@ncmir.ucsd.edu)   
+[Co-Investigator] Jeffrey Grethe (jgrethe@ncmir.ucsd.edu)   
+[Co-Investigator] Anita Bandrowski (abandrowski@ncmir.ucsd.edu)  
+[Knowledge Engineer] Tom Gillespie (tom.h.gillespie@gmail.com)   
+
+## Definitions [Optional]
+* __NIFSTD__ Neuroscience Information Framework (NIF) Standard ontology (NIFSTD) is a neuroscience
+  ontology that maintains an extensive set of terms and concepts important for the domains of
+  neuroscience and biology.
+* __InterLex__ A dynamic lexicon of biomedical terms built on the foundation of NeuroLex
```

Comment:
Also is there a project link for this as well, in case one would like to know more.

Reply:
This could be added in the documentation section below.

Reply:
Will add links below.

Reply:
NIFSTD links are already below. InterLex link resolved by 8edb24f